Open Judge System/Web/OJS.Web/Areas/Contests/Views/Submissions/View.cshtml

```diff
@@ -270,7 +277,7 @@ else
 
         if (displayCheckerComment)
         {
-            if (displayShowInputButton)
+            if (displayShowInputButton && testResult.ResultType != TestRunResultType.CorrectAnswer)
```

Comment:
Why not integrate the second check into the variable itself?

Reply:
Done